src/core/lombok/bytecode/PoolConstantsApp.java

```diff
@@ -66,7 +66,7 @@
 		CmdReader<CmdArgs> reader = CmdReader.of(CmdArgs.class);
 		CmdArgs args;
 		try {
-			args = reader.make(raw.toArray(new String[0]));
+			args = reader.make(raw.toArray(new String[raw.size()]));
```

Comment:
Hi,

Actually I'm also confusing like authors of this articles, so I've decided to check it myself and run this benchmarks on my machine:

```
Benchmark                             Mode  Cnt      Score       Error   Units
ToArrayBench.defensive_copy          thrpt   20  70353.047 ±  5513.834  ops/ms
ToArrayBench.simple_toArray          thrpt   20  92047.053 ± 10031.130  ops/ms
ToArrayBench.sized_array_fixed_size  thrpt   20  12923.122 ±   811.063  ops/ms
ToArrayBench.sized_array_from_list   thrpt   20  32272.962 ±  1582.431  ops/ms
ToArrayBench.zero_sized_array        thrpt   20  40815.625 ±  4311.759  ops/ms
```

32272.962 vs 40815.625 and `ToArrayBench.sized_array_from_list` is winner. I don't see a performance drop.